pkg/util/kube/utils.go

```diff
@@ -28,6 +28,14 @@ import (
 	corev1listers "k8s.io/client-go/listers/core/v1"
 )
 
+// LabelValue looks up and returns the value for a label on the object
+func LabelValue(objMeta metav1.Object, labelName string) string {
+	if value, exists := objMeta.GetLabels()[labelName]; exists {
```

Comment:
Given that labels are `map[string]string` and the zero value of any undefined key is the empty string, do we need this function?

Reply:
removed